web/packages/teleterm/src/ui/boot.tsx

```diff
@@ -47,8 +47,30 @@ async function boot(): Promise<void> {
   }
 }
 
+/**
+ * getElectronGlobals retrieves privileged APIs exposed through the contextBridge from preload.ts.
+ *
+ * It also immediately removes them from the window object so that if an attacker gets to execute
+ * arbitrary JS on the page, they don't get easy access to those privileged APIs.
+ */
 async function getElectronGlobals(): Promise<ElectronGlobals> {
-  return await window['electron'];
+  const globals = await window['electron'];
+  const globalsCopy = { ...globals };
```

Comment:
TIL you can spread (potentially) `undefined` values

Reply:
What do you mean by undefined values?